### DISCUSSION TITLE
bad object for metadata backup

### DISCUSSION THREAD

**Discussion by solracsf** [juicedata/juicefs#1600] [General]
I can observe **hundreds of thousands** of lines like these in the log, should I be worried about?

```
<WARNING>: bad object for metadata backup s/0/0/110_0_224297: length 18 [backup.go:143]
```

JuiceFS 1.0.0-beta2 on Redis 6.2.2

**Comment by davies** [comment]
This looks like a bug, JuiceFS listed all the objects under  chunks/ and treat it as backup of meta, we will look into this one.

**Reply by davies** [reply]
This bug is fixed by #1616
